--- tests-magic/foreach.expected.cpp	2014-09-08 01:59:57.652142765 +0200
+++ tmp/foreach.cpp	2014-09-08 02:00:23.567888541 +0200
@@ -16,8 +16,33 @@
 	}
 #line 3 "tests-magic/foreach.tmpl"
 	virtual void render(std::string ma) {
+#line 5 "tests-magic/foreach.tmpl"
+	    out() << " \n\t\t\t<!-- cut magic begin -->\n\t\t\t";
+#line 5 "tests-magic/foreach.tmpl"
+	    if ((content.container).begin() != (content.container).end()) {
+#line 6 "tests-magic/foreach.tmpl"
+		out() << "\n\t\t\t";
+#line 6 "tests-magic/foreach.tmpl"
+		for (CPPCMS_TYPEOF((content.container).begin())item_ptr =
+		     (content.container).begin(), item_ptr_end =
+		     (content.container).end(); item_ptr != item_ptr_end;
+		     ++item_ptr) {
+#line 6 "tests-magic/foreach.tmpl"
+		    CPPCMS_TYPEOF(*item_ptr) const &item = *item_ptr;
+#line 6 "tests-magic/foreach.tmpl"
+		    if (item_ptr != (content.container).begin()) {
+#line 7 "tests-magic/foreach.tmpl"
+			out() << ", \n\t\t\t";
+#line 7 "tests-magic/foreach.tmpl"
+		    }		// end of separator
+#line 6 "tests-magic/foreach.tmpl"
+		    out() << "item->name";
+#line 6 "tests-magic/foreach.tmpl"
+		}		// end of item
+#line 6 "tests-magic/foreach.tmpl"
+	    }
 #line 9 "tests-magic/foreach.tmpl"
-	    out() << " \n\n\n\n\n\n\t\t";
+	    out() << "\n\t\t\t<!-- cut magic end --> \n\t\t";
 #line 9 "tests-magic/foreach.tmpl"
 	}			// end of template render
 #line 10 "tests-magic/foreach.tmpl"
