--- tests-magic/format.expected.cpp	2014-09-04 18:23:23.948154726 +0200
+++ tmp/format.cpp	2014-09-04 18:23:25.427048254 +0200
@@ -1,3 +1,4 @@
+#include <boost/format.hpp>
 #line 1 "tests-magic/format.tmpl"
 namespace myskin {
 #line 2 "tests-magic/format.tmpl"
@@ -16,27 +17,48 @@
 	}
 #line 3 "tests-magic/format.tmpl"
 	virtual void render() {
+#line 5 "tests-magic/format.tmpl"
+	    out() << " \n\t\t\tformat with using \n\t\t\t";
+#line 5 "tests-magic/format.tmpl"
+	    out() << cppcms::filters::
+		escape((boost::format("price: %.02f") %
+			(content.item->price)).str());
 #line 6 "tests-magic/format.tmpl"
-	    out() << " \n\t\t\tformat with using \nmagic code here\n\t\t";
+	    out() << " <!-- magic -->\n\t\t";
 #line 6 "tests-magic/format.tmpl"
 	}			// end of template render
 #line 7 "tests-magic/format.tmpl"
 	virtual void render2(std::string koty, blahblah foo) {
+#line 9 "tests-magic/format.tmpl"
+	    out() << "  \n\t\t\trformat\n\t\t\t";
+#line 9 "tests-magic/format.tmpl"
+	    out() << (boost::format("blah")).str();
 #line 10 "tests-magic/format.tmpl"
-	    out() << "  \n\t\t\trformat\nmagic code here\n\t\t";
+	    out() << " <!-- magic -->\n\t\t";
 #line 10 "tests-magic/format.tmpl"
 	}			// end of template render2
 #line 11 "tests-magic/format.tmpl"
 	virtual void show_list(person_t const &person) {
+#line 13 "tests-magic/format.tmpl"
+	    out() << " \n\t\t\tformat with using\n\t\t\t";
+#line 13 "tests-magic/format.tmpl"
+	    out() << cppcms::filters::
+		escape((boost::format("name = %s, age = %03d") %
+			(cppcms::filters::downcase(person.name)) %
+			(person.age)).str());
 #line 14 "tests-magic/format.tmpl"
-	    out() << " \n\t\t\tformat with using\nmagic code here\n\t\t";
+	    out() << " <!-- magic -->\n\t\t";
 #line 14 "tests-magic/format.tmpl"
 	}			// end of template show_list
 #line 15 "tests-magic/format.tmpl"
 	virtual void show_numbers(int x, int y) {
+#line 17 "tests-magic/format.tmpl"
+	    out() << "\n\t\t\trformat with one using \n\t\t\t";
+#line 17 "tests-magic/format.tmpl"
+	    out() << (boost::format("a = %03d, b = %02d, c = \"%.03f\"") %
+		      (x) % (y) % (content.z)).str();
 #line 18 "tests-magic/format.tmpl"
-	    out() <<
-		"\n\t\t\trformat with one using \nmagic code here\n\t\t";
+	    out() << " <!-- magic -->\n\t\t";
 #line 18 "tests-magic/format.tmpl"
 	}			// end of template show_numbers
 #line 19 "tests-magic/format.tmpl"
